vm/src/stdlib/io.rs

```diff
@@ -95,6 +95,10 @@ fn io_base_cm_exit(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(vm.get_none())
 }
 
+fn io_base_flush(_zelf: PyObjectRef, _vm: &VirtualMachine) -> () {
+    ()
```

Comment:
Is this meant to do nothing? Also you can remove the return type, by default it's `()`

Reply:
The only thing this base method is meant to do is check if it's closed and if so then raise a ValueError. But as IOBase doesn't yet have a `close` method or attribute, there is currently nothing for `flush` to do. Subclasses are meant to override this and do a real flush.